src/main/scala/org/apache/mesos/chronos/scheduler/jobs/Jobs.scala

```diff
@@ -101,7 +103,8 @@ case class ScheduleBasedJob(
                              @JsonProperty override val environmentVariables: Seq[EnvironmentVariable] = List(),
                              @JsonProperty override val shell: Boolean = true,
                              @JsonProperty override val arguments: Seq[String] = List(),
-                             @JsonProperty override val softError: Boolean = false)
+                             @JsonProperty override val softError: Boolean = false,
+                             @JsonProperty override val dataJob: Boolean = false)
   extends BaseJob
 
```

Comment:
My main concern here is in regards to naming. I think it should still be a field of Job, but the name "dataProcessingType" or "dataProcessingJobType" is more descriptive to me than "dataJob"